### PR TITLE
docs(components): refine side-drawer page

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/side-drawer/README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 </script>
 ```
 
-## Open
+## Properties
```
Comment (rachelbt) [inline libs/components/src/lib/side-drawer/README.md:9]: 🦾

**Review by rachelbt — COMMENTED** [review]
LGTM
I'm just wondering wether we should wrap the "app-content" in code preview with layout for spacing or it will make the code snippet bloated.
WDYT?

File: libs/components/src/lib/side-drawer/README.md (hunk 1 above)
Comment (rinaok) [reply libs/components/src/lib/side-drawer/README.md:9]: @rachelbt Added layout

File: apps/docs/_data/components.json [hunk 2]
```diff
@@ -102,7 +102,8 @@
       "/assets/modules/components/sidenav-item/index.js",
       "/assets/modules/components/icon/index.js",
       "/assets/modules/components/text/index.js",
-      "/assets/modules/components/button/index.js"
+      "/assets/modules/components/button/index.js",
+      "/assets/modules/components/layout/index.js"
```
Comment (yinonov) [inline apps/docs/_data/components.json:106]: you don't use layout explicitly in your doc. no reason to import it here

File: apps/docs/_data/components.json [hunk 3]
```diff
@@ -102,7 +102,8 @@
       "/assets/modules/components/sidenav-item/index.js",
       "/assets/modules/components/icon/index.js",
```
Comment (yinonov) [inline apps/docs/_data/components.json:103]: you don't use icon explicitly in your docs. button already side-effect-imports it

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/_data/components.json (hunk 2 above)
Comment (rinaok) [reply apps/docs/_data/components.json:106]: @yinonov I'm using layout

File: apps/docs/_data/components.json (hunk 3 above)
Comment (rinaok) [reply apps/docs/_data/components.json]: @yinonov removed

File: apps/docs/_data/components.json (hunk 2 above)
Comment (yinonov) [reply apps/docs/_data/components.json:105]: where?

Comment (rinaok) [reply apps/docs/_data/components.json:105]: @yinonov inside the side-drawer, see @rachelbt's comment: https://github.com/Vonage/vivid-3/pull/428#pullrequestreview-993012153

File: libs/components/src/lib/side-drawer/README.md [hunk 4]
```diff
@@ -181,14 +207,17 @@ Use the `position` attribute to set the side of the side drawer.
```
Comment (rachelbt) [inline libs/components/src/lib/side-drawer/README.md:207]: the Top Bar should be:
### Top-Bar

**Review by rachelbt — COMMENTED** [review]
I think you need to update the ui png's

**Pull request merged**